substrate/in-instructions/primitives/src/shorthand.rs

```diff
@@ -0,0 +1,42 @@
+use scale::{Encode, Decode, MaxEncodedLen};
+use scale_info::TypeInfo;
+
+#[cfg(feature = "std")]
+use serde::{Serialize, Deserialize};
+
+use sp_core::{ConstU32, bounded::BoundedVec};
+
+use serai_primitives::{NativeAddress, Coin, Amount};
+
+use crate::{MAX_DATA_LEN, ExternalAddress, ExternalInInstruction, OutInstruction};
+
+#[derive(Clone, PartialEq, Eq, Debug, Encode, Decode, MaxEncodedLen, TypeInfo)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum Shorthand {
+  Raw(BoundedVec<u8, ConstU32<{ MAX_DATA_LEN }>>),
```

Comment:
Why would we ever map Raw?

Reply:
If you wanted to mint sriBTC without swapping or adding it.

Reply:
Why not a specific mint call then?

Reply:
Mint isn't meaningfully smaller if we make a dedicated call, does increase our complexity, and if it replaces our exhaustive option, potentially leaves us incomplete.

Reply:
We should be verbose, Raw tells us nothing.

Reply:
It's actually going to be `Shorthand::Raw(InInstruction::Transfer(serai_address))`. I do agree Raw isn't exactly descriptive there... We could offer a function `Shorthand::transfer(SeraiAddress)` to construct that?